packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -253,6 +272,47 @@ export default class Meeting extends StatelessSparkPlugin {
     this.correlationId = id;
   }
 
+  startAudioVideo(mediaDirection) {
+    this.audio.init({mute: !mediaDirection.sendAudio, self: true}, this);
+    this.video.init({mute: !mediaDirection.sendVideo, self: true}, this);
+  }
+
+  muteAudio() {
+    const options = {};
+    options.mute = true;
+    options.self = true;
+    this.toggleAudio(options);
+  }
+
+  unmuteAudio() {
+    const options = {};
+    options.mute = false;
+    options.self = true;
+    this.toggleAudio(options);
+  }
+
+  muteVideo() {
+    const options = {};
+    options.mute = true;
+    options.self = true;
+    this.toggleVideo(options);
+  }
+
+  unmuteVideo() {
```

Comment:
I like the syntax that is exposed to the user here.

Reply:
👍